tests/integration/member_test.go

```diff
@@ -115,3 +118,31 @@ func TestSnapshotAndRestartMember(t *testing.T) {
 		}
 	}
 }
+
+func TestRemoveMember(t *testing.T) {
+	integration.BeforeTest(t)
+	c := integration.NewCluster(t, &integration.ClusterConfig{Size: 3, UseBridge: true, BackendBatchInterval: 1000 * time.Second})
+	defer c.Terminate(t)
+	// membership changes additionally require cluster to be stable for etcdserver.HealthInterval
+	time.Sleep(etcdserver.HealthInterval)
+
+	err := c.RemoveMember(t, c.Client(2), uint64(c.Members[0].ID()))
+	require.NoError(t, err)
+	// membership changes additionally require cluster to be stable for etcdserver.HealthInterval
```

Comment:
Hmm there are no membership changes after, I think this is a linearizability issue.

Reply:
Removing this wait because `RemoveMember` already `WaitMembersMatch`